apollo-client/src/main/java/com/ctrip/framework/apollo/internals/ConfigServiceLocator.java

```diff
@@ -160,8 +168,14 @@ private String getDeprecatedCustomizedConfigService() {
    * @return the services dto
    */
   public List<ServiceDTO> getConfigServices() {
+    initialize();
     if (m_configServices.get().isEmpty()) {
-      updateConfigServices();
+      // quick fail
+      throw new ApolloConfigException(
+          "No available config service, "
+              + "server side maybe crash or network cannot connect to server from this ip, "
+          + "one of meta service url is " + assembleMetaServiceUrl()
+      );
```

Comment:
Is it the same if we keep all the logic unchanged and only remove the `updateConfigServices` call and throw the exception here?

Reply:
Done.